packages/upward-js/lib/ParameterValidator.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Return functions which return the empty string if validation passes, and a
+ * descriptive string if validation fails.
+ */
+const typeTests = {
+    oneOf(...allowed) {
+        return value =>
+            allowed.includes(value)
+                ? undefined
+                : `must be one of: ${allowed.join()}`;
```

Comment:
In the passing case, why does this return `undefined`, rather than `''` like all the rest of these tests?